WordPress/Classes/Models/Product.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import StoreKit
+
+@objc
+protocol Product {
+    var localizedDescription: String { get }
+    
+    var localizedTitle: String { get }
+    
+    var price: NSDecimalNumber { get }
+    
+    var priceLocale: NSLocale { get }
+    
+    var productIdentifier: String { get }
+}
+typealias Products = [Product]
```

Comment:
I'm not sure this type alias is really necessary... I think we should probably just use `[Product]` instead?